src/main/kotlin/fi/metatavu/oss/api/impl/devicesurveys/DeviceSurveyRepository.kt

```diff
@@ -75,10 +75,11 @@ class DeviceSurveyRepository: AbstractRepository<DeviceSurveyEntity, UUID>() {
      * @return list of device surveys
      */
     suspend fun listDeviceSurveysToPublish(): List<DeviceSurveyEntity> {
-        val queryString = "status = :status AND publishStartTime <= :publishStartTime"
+        val queryString = "status = :status AND publishStartTime <= :publishStartTime AND publishEndTime > :publishEndTime"
```

Comment:
Should this query take account publishStartTime and publishEndTime fields with null values or do these always have some value?